api/v1beta1/virtinkmachine_types.go

```diff
@@ -1,8 +1,10 @@
 package v1beta1
 
 import (
-	virtv1alpha1 "github.com/smartxworks/virtink/pkg/apis/virt/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cdiv1beta1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
+	virtv1alpha1 "github.com/smartxworks/virtink/pkg/apis/virt/v1alpha1"
```

Comment:
We can treat Virtink as an outside lib just like CDI in capch, so there's no need to add a blank line above.